.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,14 +15,14 @@
     explained in https://github.com/eslint/eslint/pull/12848#issuecomment-580302888
```

Comment:
This is intentional. Pls check #12896

Reply:
Fair enough, but annoying as a contributor